.changeset/spotty-pants-walk.md

```diff
@@ -0,0 +1,6 @@
+---
+"@salt-ds/lab": minor
+---
+
+Remove `Slider` props, `pageStep`, `pushable`, `pushDistance`, `disabled`, `marks`, `hideMarks`, `hideMarkLabels`
+Remove `Slider` range feature
```

Comment:
Range feature to follow - added this incase we want to do a release before the range slider is ready